BrowserKit/Sources/Shared/Prefs.swift

```diff
@@ -257,6 +257,15 @@ public struct PrefsKeys {
     public struct Usage {
         public static let profileId = "profileId"
     }
+
+    public struct PrivacyNotice {
+        // Timestamp in milliseconds for when the privacy notice homepage card was last shown
+        public static let notifiedDate = "NotifiedDate"
```

Comment:
just to lessen the chance for conflicts, I'd update this to something like "PrivacyNotice.NofifiedDate" or something.